### PR TITLE
gather-extra: update labels for request latency metrics

### PR DESCRIPTION
**Pull request by vrutkovs**
In 4.13 `rest_client_request_duration_seconds_bucket` is using `host` instead of `url`

### CODE REVIEW

**Comment by vrutkovs** [comment]
cc @dgrisonnet

File: ci-operator/step-registry/gather/extra/gather-extra-commands.sh [hunk 1]
```diff
@@ -487,17 +487,17 @@ ${t_all}     cluster:promtail:failed_targets   sum by (pod) (promtail_targets_fa
...
+${t_all}     cluster:rest:client:requests:error:total:rate sum by(type) (label_replace(sum(rate(rest_client_requests_total{code="<error>",host=~"api-int.*"}[${d_all}])) / sum(rate(rest_client_requests_total{host=~"api-int.*"}[${d_all}])),"type","load_balancer","","") or label_replace(sum(rate(rest_client_requests_total{code="<error>",host!~"(api-int|\\[::1\\]|127\\.0\\.0\\.1|localhost).*"}[${d_all}])) / sum(rate(rest_client_requests_total{host!~"(api-int|\\[::1\\]|127\\.0\\.0\\.1|localhost).*"}[${d_all}])),"type","service","","") or label_replace(sum(rate(rest_client_requests_total{code="<error>",host=~"(\\[::1\\]|127\\.0\\.0\\.1|localhost).*"}[${d_all}])) / sum(rate(rest_client_requests_total{host=~"(\\[::1\\]|127\\.0\\.0\\.1|localhost).*"}[${d_all}])),"type","pod","",""))
```
Comment (dgrisonnet) [inline ci-operator/step-registry/gather/extra/gather-extra-commands.sh:498]: good catch

**Review by dgrisonnet — APPROVED** [review]
/lgtm

**Pull request merged**